ml-agents/mlagents/trainers/stats.py

```diff
@@ -224,12 +228,15 @@ def __init__(self, base_dir: str, clear_past_data: bool = False):
         self.summary_writers: Dict[str, SummaryWriter] = {}
         self.base_dir: str = base_dir
         self._clear_past_data = clear_past_data
+        self.hidden_keys: List[str] = hidden_keys if hidden_keys else []
```

Comment:
```suggestion
        self.hidden_keys: List[str] = hidden_keys if hidden_keys is not None else []
```

I think in some cases the bool value of None is ambiguous